src/state/transactions/hooks.tsx

```diff
@@ -129,3 +129,24 @@ export function useHasPendingApproval(tokenAddress?: string, spender?: string):
   }
   return hasPendingApproval
 }
+
+// returns whether a account has a pending claim transaction
+export function useHasPendingClaim(auctionId?: number, from?: string | null): boolean {
+  const allTransactions = useAllTransactions()
+
+  return useMemo(() => {
+    return (
+      typeof auctionId === 'number' &&
+      typeof from === 'string' &&
```

Comment:
From is never used, is it?

Reply:
I forgot to put `from` in the statement, for the sake of context. But it would not be necessary since that `useAllTransactions` brings them filtered.